doc/Caliptra.md

```diff
@@ -415,21 +415,171 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
+
+For IDevID to endorse LDevID, Caliptra requires the vendor to implement the following field values for the IDevID certificate:
+| Field                          | Sub Field   | Value
+| -------------                  | ---------   | ---------
+| Version                        | v3          | 2
+| Serial Number                  |             | First 20 bytes of SHA256 hash of IDevID public key
+| Validity                       | notAfter    | 99991231235959Z
+| Subject Name                   | CN          | Caliptra IDevID
+|                                | serialNumber | SHA384 hash of Subject public key
+| Subject Public Key Info        | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+|                                | Public Key  | DPE Alias Public Key value
+| Signature Algorithm Identifier | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+| Signature Value                |             | Digital signature for the certificate
+| KeyUsage                       | keyCertSign | 1
+| Basic Constraints              | CA          | TRUE
+|                                | pathLen     | 5
+
+Caliptra does not consume the IDevID certificate. Caliptra needs attributes of the IDevID certificate in order to generate the Authority Key Identifier extension for the LDevID and to populate the TCG Universal Entity ID (UEID) extension for Caliptra-generated certificates. The vendor must fuse these attributes into the IDevID attribute fuses for Caliptra to consume. The encoding of these attribute fuses are as follows:
+* Flags (byte 0, bits [1:0]): Key ID algorithm for IDevID Subject Key Identifier.
+** 0 = SHA1 of IDevID public key
+** 1 = truncated SHA256 of IDevID public key
+** 2 = truncated SHA384 of IDevID public key
+** 3 = raw
+* Subject Key ID (bytes 4 to 23): if Flags = 3, the IDevID Subject Key Identifier to use as the LDevID Authority Key Identifier.
+* Manufacturer Serial Number (bytes 24 to 31): the per-device-unique serial number to be used for the TCG UEID extension in the Caliptra-generated LDevID, Alias<sub>FMC</sub>, and Alias<sub>RT</sub> certificates.
+
+The IDevID certificate is per-device-unique and non-renewable. The SoC must be able to retrieve the IDevID certificate at runtime. To save flash space and aid in recoverability, it is recommended that the vendor define an IDevID certificate template such that the SoC at runtime can reconstruct the same certificate that the pCA endorsed. The SoC is recommended to store the IDevID certificate signature in fuses and the IDevID certificate template in the firmware image. Caliptra runtime firmware provides APIs to aid in reconstructing the certificate:
```

Comment:
`per-device-unique and non-renewable`-->`the unique serial number of the device and is non-renewable`